eth-connector/src/lib.rs

```diff
@@ -693,8 +832,83 @@ impl FungibleTokenReceiver for EthConnectorContract {
 
 #[cfg(test)]
 mod tests {
+
     use super::*;
 
+    use near_sdk::test_utils::VMContextBuilder;
+    use near_sdk::testing_env;
```

Comment:
We use our crate [aurora-workspace-eth-connector](https://github.com/aurora-is-near/aurora-workspace/tree/main/workspace-eth-connector) for testing eth-connector. You can find examples of usage in the code.

Reply:
Added tests here using `aurora-workspace-eth-connector`. PR: https://github.com/aurora-is-near/aurora-workspace/pull/29